### PR TITLE
Added files from from previous project

### PR DESCRIPTION
**Pull request by Elisha-Tetteyfio**
Creating a pull request anf uploading files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  
+      <h1 id="title">Microverse Trial Collaboration Survey Form</h1>
+      <form id="form">
+        <div class="form-control">
+          <label for="name" id="name-label">
+            Name
+          </label>
```
Comment (Elisha-Tetteyfio) [inline index.html:18]: [OPTIONAL] I think writing simple codes on one line makes your work shorter and easier to read

File: index.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+          <input type="text"
+                  id="name"
+                  placeholder="Enter your name" />
```
Comment (Elisha-Tetteyfio) [inline index.html:21]: [OPTIONAL] All this could have also been on a single line

File: index.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+                   Professional
+               </option>
+               <option value="other">Other</option>
+               </select>
+          </div>
+
+          <div class="form-control">
+               <label>
+                   Would you recommed Microverse to a friend?
+               </label>
```
Comment (Elisha-Tetteyfio) [inline index.html:61]: Label tags are best used for inputs. As this is not an input, I would suggest using a <h4> or <h5> or any other appropriate tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+             <label for="inp-6">
+                 <input type="checkbox"
+                        name="inp">JavaScript</input></label>
+             <label for="inp-7">
+                 <input type="checkbox"
+                        name="inp">React</input></label>
+             <label for="inp-7">
+                 <input type="checkbox"
+                        name="inp">Angular</input></label>
+             <label for="inp-7">
```
Comment (Elisha-Tetteyfio) [inline index.html:110]: Kindly make a reference for your labels for inp-1 to inp-7 as they don't seem to be referencing any input element

**Review by Elisha-Tetteyfio — COMMENTED** [review]
Great job overall. Your project is almost ready to be merged

File: style.css [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+.form-control textarea {
+    border: 1px solid #777;
+    border-radius: 2px;
+    font-family: inherit;
+    padding: 10px;
+    display: block;
+    width: 95%;
+    height: 5em;
+}
+
```
Comment (koomekevo) [inline style.css:34]: @Elisha-Tetteyfio You should have highlighted that the student need not leave huge lines of space between codes. It makes the code difficult to read and also untidy.

File: index.html (hunk 4 above)
Comment (koomekevo) [reply index.html:110]: You should have phrased this comment in this manner: _Kindly make a reference for your labels for inp-1 to inp-7 as they don't seem to be referencing any input element_

**Review by koomekevo — CHANGES_REQUESTED** [review]
Hi @Elisha-Tetteyfio, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
:heavy_check_mark: You have made a review with at least 2 comments. 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
```
Comment (Elisha-Tetteyfio) [inline style.css:35]: Please reduce the amount of space you leave between code to make reading it easier

**Comment by Elisha-Tetteyfio** [comment]
Add the needed comments

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @Elisha-Tetteyfio,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**